example/index.ios.js

```diff
@@ -1,33 +1,34 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
-
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import {
+  Alert,
   AppRegistry,
   StyleSheet,
-  Text,
-  View
-} from 'react-native';
+  Button,
+  View,
+} from 'react-native'
+import { TPSTwitterModule } from 'tipsi-twitter'
+
+TPSTwitterModule.init({
+  consumerKey: 'T2VS8tuBEOMBO604qSkg',
+  consumerSecret: 'yB8RTQUoUvgcQb0DpSXRIcW2GX8aymjFDnQVYMGCo',
```

Comment:
This keys shouldn't be used in example application. Consider creating/using other pair